tests/modeling.py

```diff
@@ -0,0 +1,185 @@
+"""
+Tests for dessia_common.modeling file
+
+"""
+import numpy as npy
+from sklearn import linear_model, tree, ensemble, svm, neural_network
+
+from dessia_common.models import all_cars_no_feat
+from dessia_common.datatools.dataset import Dataset
+from dessia_common.datatools.modeling import StandardScaler, IdentityScaler, Ridge, SupportVectorRegressor,\
+    SupportVectorClassifier, MLPRegressor, DecisionTreeRegressor, DecisionTreeClassifier, RandomForestRegressor, \
+        RandomForestClassifier, MLPClassifier, Scaler, Model, LinearModel, Tree, RandomForest, SupportVectorMachine, \
+            MultiLayerPerceptron, LinearRegression
+
+
```

Comment:
Too complicated to understand, should write a simple script to understand how to use, maybe on ony one classfier